content/en/blog/_posts/2022-10-03-current-state-2019-third-party-audit.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: blog
+title: "Current State: 2019 Third Party Security Audit of Kubernetes"
+date: 2022-10-03
+slug: current-state-2019-third-party-audit
+---
+
+**Authors**: Cailyn Edwards, Pushkar Joglekar, Rory McCune and Rey Lejano
+
+A brand new Third Party Security Audit of Kubernetes, is expected to be
+published later this month (Oct 2022). 
+
+In preparation for that, let's look at the state of findings that were made
+public as part of the last [third
+party security audit of
+2019](https://github.com/kubernetes/sig-security/tree/main/sig-security-external-audit/security-audit-2019).
+
+## Motivation
+
+Craig Ingram (@cji) has graciously attempted over the years to keep track of the 
+status of the findings reported in the last audit in this issue:
+[kubernetes/kubernetes#81146](https://github.com/kubernetes/kubernetes/issues/81146)
+. This blog post will attempt to dive deeper into this, address any gaps
+in tracking and become a point in time summary of the state of the
+findings reported from 2019.
+
+This blog should also help folks gain confidence through transparent
```

Comment:
```suggestion
This blog should also help folks gain confidence in Kubernetes security through transparent
```
just trying to clarify this a bit - are we saying folks should feel confident in Kubernetes security because we are working on fixes to security issues?

Reply:
This tries to address only improvements that came out of the security audit.

Reply:
Confidence in kubernetes security overall, would be probably a bit of a reach :D